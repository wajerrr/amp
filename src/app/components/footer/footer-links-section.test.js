import React from 'react';
import renderer from 'react-test-renderer';
import FooterLinksSection from './footer-links-section';

const section = {
  id: 'menu-unicorn-primary-user-action',
  type: 'menu',
  hasPart: {
    parts: [
      {
        headline: 'Subscribe',
        url: {
          canonical: 'https://www.economist.com/subscribe',
        },
        meta: '',
        isPartOf: {
          context: {
            position: 0,
          },
        },
      },
      {
        headline: 'Contact us',
        url: {
          canonical: 'https://www.economist.com/contact-info',
        },
        meta: '',
        isPartOf: {
          context: {
            position: 1,
          },
        },
      },
    ],
  },
};

describe('FooterLinksSection', () => {
  it('should match the snapshot', () => {
    const footer = renderer
      .create(<FooterLinksSection menuItem={section} sectionLevel={1} />)
      .toJSON();
    expect(footer).toMatchSnapshot();
  });
});
