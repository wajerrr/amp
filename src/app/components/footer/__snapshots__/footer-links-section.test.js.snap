// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterLinksSection should match the snapshot 1`] = `
Array [
  <a
    className="css-11i4prc-StyledFooterLink e1g1liug0"
    href="https://www.economist.com/subscribe"
  >
    Subscribe
  </a>,
  <a
    className="css-11i4prc-StyledFooterLink e1g1liug0"
    href="https://www.economist.com/contact-info"
  >
    Contact us
  </a>,
]
`;
